sites/hashdev/src/_pages/blog/10_error-stack-update-0-2.mdx

```diff
@@ -0,0 +1,185 @@
+- `Termination` has been implemented for `Report`s: it will return the `ExitCode` provided by any `Context` if present, or `ExitCode::FAILURE` will be returned.
+- Compatibility for `anyhow` and `eyre` has been added to convert their types into `Report`.
+  - We’ve provided a trait to convert `Result<_, anyhow::Error>` or `Result<_, eyre::Report>` to `Result<_, Report<anyhow::Error>>` or `Result<_, Report<eyre::Report>>`.
+  - The additional sources of `anyhow::Error` and `eyre::Report` are also attached as frames to maintain their history.
+
+## Breaking changes
+
+We care a lot about breaking changes and will always try to conform to [SemVer](https://semver.org). A few principles guide our approach:
+
+- we’re not afraid to make breaking changes if it will improve the usability and functionality of the library.
+- we try to maintain consistency. E.g. we’ve deprecated `Report::span_trace` to avoid special casing `SpanTrace` and keep the API the same as for backtraces.
+- we try to avoid duplication. If one feature is superseded by another, we’ll deprecate the old one and remove it later, as with the new hook system.
+
+Version 0.2 of `error-stack` introduces these breaking changes and deprecations:
+
+- The minimal supported Rust version was bumped to 1.63 to greatly simplify maintenance.
+- We switched to the upstream implementation of `Provider`, so if you were on nightly and used our provider API, you need to change the imports to `core::any::`.
+- Remove the unused features `hooks`, `futures`, and `futures-core`
+- `Report::backtrace` (and `Report::span_trace` for consistency) were deprecated because you cannot get `Backtrace` from `Error` anymore.
+  - non-nightly channels should use `Report::downcast_ref::<Backtrace>` / `Report::downcast_ref::<SpanTrace>`.
+  - nightly channels should use`Report::requested_ref::<Backtrace>` / `Report::requested_ref::<SpanTrace>`.
+- The format of the `Debug` output has changed significantly. If you had a dependency on the format of the `Debug` output (which we don’t recommend), it might break.
+- We deprecated `IntoReport::report`. Please use `IntoReport::into_report` instead.
+- We deprecated the following as part of the changes we made to how hooks work:
+  - instead of `Report::set_debug_hook` please use `Report::install_debug_hook`.
+  - Please don’t use `Report::set_display_hook`.
+- Because multiple error sources are now possible, we deprecated:
+  - `Frame::source`; please use `Frame::sources` instead.
+  - `Frame::source_mut`; please use `Frame::sources_mut` instead.
+
+## What’s next?
+
+We’ve got a lot of ideas! Among them are: adding support for `serde` to provide a hook-based interface to serialize `Report`s; adding support for `defmt`; adding support for WASM environments; and continuing to iterate on the ergonomics of how we support related errors and multiple sources. Please let us know what you think.
```

Comment:
Optional bulleting

Where should people let us know? Can we link somewhere?
```suggestion
We’ve got a lot of ideas! Among them are:
- adding support for `serde` to provide a hook-based interface to serialize `Report`s.
- adding support for `defmt`; adding support for WASM environments.
- and continuing to iterate on the ergonomics of how we support related errors and multiple sources. 

Please let us know what you think!
```

Reply:
I think bullets are more readable.

I think the "Getting Involved" section below is close enough to this section, or what do you think?

Reply:
5be57d54f25ad98d028464178618020e3c0c3c33